android/app/src/main/java/com/reactnativenavigation/core/objects/Screen.java

```diff
@@ -68,15 +73,18 @@ public Screen(ReadableMap screen) {
         navigatorId = getString(screen, KEY_NAVIGATOR_ID);
         navigatorEventId = getString(screen, KEY_NAVIGATOR_EVENT_ID);
         icon = getInt(screen, KEY_ICON);
+        if(screen.hasKey(KEY_PROPS))
```

Comment:
Add Braces to conditional statement. We follows google's [code style conventions](https://source.android.com/source/code-style.html).